test/services/WelshTextBuilderSpec.scala

```diff
@@ -54,185 +48,106 @@ class TextBuilderSpec extends BaseSpec {
     val answerWithNoHint = Phrase("Yes", "Welsh, Yes")
     val answerWithHint = Phrase("Yes[hint:You agree with the assertion]", "Welsh, Yes[hint:Welsh, You agree with the assertion]")
 
-    val answer = Text("Yes", "Welsh, Yes")
-    val hint = Text("You agree with the assertion", "Welsh, You agree with the assertion")
+    val answer = Text("Yes")
+    val hint = Text("You agree with the assertion")
   }
 
   "TextBuilder placeholder parsing" must {
 
     "Convert label reference with default output format placeholders within phrase to LabelRef TextItems" in new Test {
```

Comment:
As with the English text builder some of these tests need the English and Welsh text to differ.

Reply:
updated